docs/index.rst

```diff
@@ -1,17 +1,67 @@
 .. include:: ../README.rst
    :end-before: About
 
+
 .. include:: ../README.rst
-   :start-after: -----
+   :start-line: 25
    :end-before: Features
 
-.. toctree::
-   :numbered:
 
-   introduction
-   cli
-   restapi
-   contributing
-   changes
-   license
-   authors
+.. include:: ../README.rst
+   :start-line: 32
+   :end-before: Useful links
+
+**Contents**
+
+.. contents::
+   :local:
+   :depth: 1
+   :backlinks: none
+
+
```

Comment:
Before CLI API, you can introduce Installation section, something like:

```
Installation
---------------

Please follow detailed _REANA cluster installation guide_.
```

pointing to docs.reana.io.  The text could be the same for all cluster components.